Gemfile

```
@@ -61,9 +61,10 @@ gem "parslet"
 gem "pg"
 gem "pg_search"
 gem "prawn-table"
-gem "puma"
+gem "puma", "~> 6.5.0"
```

```suggestion
gem "puma"
```

The version has been upgraded by dependabot https://github.com/DFE-Digital/teaching-vacancies/pull/7314/files